docs/user-guide/configuring-at-tls-for-zowe-server-single-service.md

```diff
@@ -0,0 +1,972 @@
+}
+
+TTLSEnvironmentAdvancedParms ClientEnvironmentAdvParms
+{
+  Renegotiation Disabled
+  3DesKeyCheck Off
+  ClientEDHGroupSize Legacy
+  ServerEDHGroupSize Legacy
+  PeerMinCertVersion Any
+  ServerScsv Off
+  MiddleBoxCompatMode Off
+  CertValidationMode Any
+}
+
+TTLSConnectionAdvancedParms ZoweClientNoX509ConnAdvParms
+{
+# No CertificateLabel; Keyring contains no X.509 Client Certificate
+  ApplicationControlled Off
+  SecondaryMap Off
+  SSLv3 Off
+  TLSv1 Off
+  TLSv1.1 Off
+  TLSv1.2 On
+  TLSv1.3 On
+}
+```
+
+* **`Jobname`**  
+ This parameter is defined explicitly for the API Gateway and ZAAS component and is formed with the `zowe.job.prefix` setting from `zowe.yaml` plus `AG` (API Gateway) and `AZ` (ZAAS) suffixes. Choosing `ZWE1A*` as a jobname pattern captures both servers.
+
+* **`ZoweNoX509Keyring`**  
+This parameter is used for outbound rules that do not require or prohibit X.509 Client Certificate authentication, is distinct from `ZoweKeyring`. Refer to the complete PAGENT rules in the [Full example of AT-TLS configuration](#full-example-of-at-tls-configuration).
+  
+:::
+
+### Ciphers
+
+:::note
+This list of ciphers is provided as an example only. Actual ciphers should be customized according to your specific configuration.
+:::
+
+The list of supported ciphers should be constructed according to the TLS supported versions.
+Ensure that the cipher list has matches with non-AT-TLS-aware clients.
+
+<details>
+<summary>Click to view an example of Cipher parameters.</summary>
+
+```bash
+TTLSCipherParms CipherParms
+{
+  V2CipherSuites TLS_RC4_128_WITH_MD5
+  V2CipherSuites TLS_RC4_128_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_DES_192_EDE3_CBC_WITH_MD5
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_AES_128_GCM_SHA256
+  V3CipherSuites TLS_AES_256_GCM_SHA384
+  V3CipherSuites TLS_CHACHA20_POLY1305_SHA256
+}
+```
+
+</details>
+
+## Using AT-TLS for API ML in High Availability
+
+When Zowe is configured in High Availability mode, AT-TLS configuration is mostly the same as in a single-instance deployment. The primary difference is that the AT-TLS rules must allow communication across multiple LPARs.
+
+Ensure that the `RemoteAddr` setting in the rules accounts for the following connections:
+
+* Discovery Service to Discovery Service. This is the replica request.
+* API Gateway Service to southbound services (including app-server and ZSS) running in another LPAR.
+* API Gateway Service to ZAAS running in another LPAR.
+* Southbound services to the Discovery Service. This applies during onboarding.
```

Comment:
The discovery service targetted by the southbound service for onboarding can be running in either LPAR